test/k8s/manifests/egress-nat-policy.yaml

```diff
@@ -63,12 +63,12 @@ spec:
         cilium.io/ci-node: k8s2
     egressIP: INPUT_EGRESS_IP
 ---
-apiVersion: cilium.io/v2alpha1
-kind: CiliumEgressNATPolicy
+apiVersion: cilium.io/v2
```

Comment:
This should stay `v2alpha1` as it's still using the old `egressSourceIP` field (beside that tests seem happy).
We can then remove it once we get rid of the old API